shared/src/main/scala/amf/client/render/Renderer.scala

```diff
@@ -72,12 +72,10 @@ class Renderer(val vendor: String, val mediaType: String, private val env: Optio
     generate(unit._internal, InternalRenderOptions(options)).asClient
 
   /** Asynchronously renders the syntax to a provided writer and returns it. */
-  @JSExport
```

Comment:
java Writer can't be used from JS. For JS the method with the `LimitedStringBuffer` arg should be used.